_posts/2023-10-09-towards_3.0.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Towards MDAnalysis 3.0
+---
+
+## Overview
+
+MDAnalysis is beginning to plan for our 3.0 release! Following discussions at our first [UGM][], we (the MDAnalysis [Core Developers][]) have decided on a set priorities for this exciting milestone (detailed below). Our primary goals can be divided into several main themes.
+
+
+### A trimmed down core library
+
+As MDAnalysis continues to grow along with the field of molecular simulations, we are aiming to continue to provide a highly functional, easily maintainable package that can be the foundation of a wide variety of downstream tools.
+
+Following the successful introduction of the [MDAKits ecosystem][] (modelled on the SciKit ecosystem, see the [MDAKits whitepaper][]) as part of our [EOSS4][] work, which provides infrastructure for creating [FAIR][] packages that address specific scientific questions in the analysis of molecular simulation, we are aiming to move some more specialized analyses currently present in the MDAnalysis core library to [MDAKits][]. Several analysis modules, including  hole2, encore, path-similarity, helanal and waterdynamics are being assessed to undergo this transition. Indeed some of these packages are already deployed as [MDAKits][], and are deprecated in the core library.
```

Comment:
We should point to our scipy paper (doi https://doi.org/10.25080/gerudo-f2bc6f59-00a) rather than the whitepaper

Reply:
done.